.bundler-audit.yml

```diff
@@ -4,3 +4,4 @@
 # - leave file with `ignore: []` if ignore list is empty
 ignore:
   - CVE-2024-6484 # ignore until a patch is available https://github.com/advisories/GHSA-9mvj-f7w8-pvh2
+  - CVE-2024-54133 # ignore until Rails is upgraded to >= 7.0
```

Comment:
FYI got a PR for that read here https://github.com/zendesk/samson/pull/4120
... we are almost done with hosting samson ourselves, so if you can verify it works we can merge that too